Modules/_tkinter.c

```diff
@@ -3419,8 +3419,9 @@ PyInit__tkinter(void)
 #endif
 
     Tkinter_TclError = PyErr_NewException("_tkinter.TclError", NULL, NULL);
-    if (PyModule_AddObjectRef(m, "TclError", Tkinter_TclError)) {
+    if (PyModule_AddObject(m, "TclError", Tkinter_TclError)) {
```

Comment:
`Tkinter_TclError` should be a strong reference to `_tkinter.TclError`. Even if you remove "TclError" from the the module dict, the use of `Tkinter_TclError` should not crash.

BTW, do not use `PyModule_AddObject()`, it is broken beyond repair. Use `PyModule_AddObjectRef()` or `PyModule_Add()`, what is more convenient.

Reply:
Ok, where should the strong reference to `Tkinter_TclError` be decref'd? Should we add a module clear function?